src/pages/index.tsx

```diff
@@ -17,7 +18,7 @@ import {
   Footer,
 } from '../sections';
 
-export default function IndexPage() {
+const IndexPage = () => {
   // const context = useDocusaurusContext();
   // const {
   //     customFields: { keywords },
```

Comment:
Can we delete this unused part of the code?

Reply:
Sure!